### PR TITLE
558 load sids

### PR DESCRIPTION
**Pull request by dmlyons2**

### CODE REVIEW

File: dashboard/views/chemical_curation.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    except Exception as e:
+                        print(e)
+                        pass
+
+        except Exception as e:
+            messages.error(request, "Unable to upload file. " + repr(e))
+    if records_processed > 0:
+        print(records_processed)
+        data.update({"records_processed": records_processed})
+    print(data)
```
Comment (mwfrost) [inline dashboard/views/chemical_curation.py]: @dmlyons2 Can you remove these `print()` statements? They appear in every test run.

Comment (dmlyons2) [reply dashboard/views/chemical_curation.py]: doneB

Comment (debboutr) [reply dashboard/views/chemical_curation.py]: ha, well this is pretty funny
```
        print('\nprint(some very important test output)')
        print('|￣￣￣￣￣|')
        print('|   HI      |')
        print('|           |' )
        print('|  MATT     |' )
        print('|           |')
        print('|___________|')
        print('(\__/) || ')
        print('(•ㅅ•) || ')
        print('/ 　 づ')
```

**Comment by zachmunzenrider** [comment]
One more thought on this--you might want to use the `csv.DictReader()` like we do starting on line 253 in `views/data_group.py`. It handles a lot of the field-manipulation code that's being done manually in this ticket, and as an extra benefit makes the code more readable.

**Comment by zachmunzenrider** [comment]
After discussion with Dave about the use case for this ticket, we agreed that it should pass review as-is

**Pull request closed without merging**